tests/testthat/test-rw-methods.R

```diff
@@ -25,13 +25,21 @@ test_that('readTree([w/o spans]) works', {
   expect_that(tree['ntips'], equals(4))
   expect_false(tree['wspn'])
 })
-test_that('readTree([w/ node labels]) works', {
+test_that('readTree([w/ malformed node labels]) works', {
   text <- "(A,B,(C,D)0.9)0.8;"
   tree <- readTree(text=text, spcl_slt_nm='bootstrap',
                    wndmtrx=sample(c(TRUE, FALSE), 1))
   expect_that(tree['ntips'], equals(4))
```

Comment:
Can you add this? Probably should have been there all along.
```suggestion
  expect_equal(tree['othr_slt_nms'], 'bootstrap')
  expect_true(all(c('0.9', '0.8') %in%
                    as.character(getNdsSlt(tree = tree, slt_nm = 'bootstrap',
                                           ids = tree['nds']))))
  expect_that(tree['ntips'], equals(4))
```